starport/pkg/cosmosanalysis/module/module_test.go

```diff
@@ -13,12 +13,12 @@ var testModule = Module{
 	Name: "planet",
 	Pkg: protoanalysis.Package{
 		Name:         "tendermint.planet.planet",
-		Path:         "testdata/planet/proto/planet",
-		Files:        protoanalysis.Files{protoanalysis.File{Path: "testdata/planet/proto/planet/planet.proto", Dependencies: []string{"google/api/annotations.proto"}}},
+		Path:         "testutil/planet/proto/planet",
```

Comment:
What's the reason behing changing `testdata` to `testutil`?

`testdata` is a dir known and ignored by the Go compiler. Changing to `testutil` will break this behaviour. 

Search for: https://go.dev/ref/mod > testdata

Reply:
It was accidental change by IDE refactoring feature I will revert it